packages/odyssey-storybook/src/components/odyssey-mui/Tag/Tag.stories.tsx

```diff
@@ -111,64 +118,98 @@ const storybookMeta: Meta<TagProps> = {
   args: {
     label: "Starship",
     colorVariant: "default",
+    testId: "tag-default",
   },
   decorators: [MuiThemeDecorator],
   tags: ["autodocs"],
 };
 
 export default storybookMeta;
 
+const checkTagStyles = async (
+  canvas: ReturnType<typeof within>,
+  testId: string,
+  expectedBackgroundColor: string,
+  expectedColor: string,
+) => {
+  const tag = canvas.getByTestId(testId);
+  await expect(tag).toHaveStyle(`background-color: ${expectedBackgroundColor}`);
+  await expect(tag).toHaveStyle(`color: ${expectedColor}`);
+};
+
```

Comment:
Same here, we shouldn't select by test ID, we should use accessibility selectors.

Test IDs are deprecated because they're there for existing Selenium tests.